lib/instance-validator.js

```diff
@@ -235,9 +235,12 @@ class InstanceValidator {
       } else {
         validatorFunction = Promise.promisify(validator.bind(this.modelInstance));
       }
-      return validatorFunction().catch(this._pushError.bind(this, false, errorKey));
+      return validatorFunction()
+              .catch(e => this._pushError(false, errorKey, e, optValue));
     } else {
-      return Promise.try(validator.bind(this.modelInstance, invokeArgs)).catch(this._pushError.bind(this, false, errorKey));
+      return Promise
+              .try(validator.bind(this.modelInstance, invokeArgs))
```

Comment:
> Btw, is invokeArgs an array? Because bind() doesn't take an array of arguments, it is variadic

Not its just single value from validator.args (which can be array), but its correctly passed, so no worries :)